docs/reference/scaling-policies.md

```diff
@@ -88,32 +88,53 @@ properties:
+| spec.kafkaStreamsPolicy.maxReplicas        | Specifies a constraint on the maximum number of replicas. The policy will never scale the application past this max.                                                            |
+| spec.kafkaStreamsPolicy.minReplicas        | Specifies a constraint on the minimum number of replicas. The policy will never scale below this min.                                                                           |
+| spec.kafkaStreamsPolicy.maxScaleUpReplicas | Specifies a constraint on the maximum number of replicas the policy will add in a given evaluation                                                                              |
+| spec.kafkaStreamsPolicy.diagnosers         | A list of diagnosers that govern the policy's behaviour. Diagnosers can produce conflicting actions. In that case, the diagnoser that comes first in the list takes precedence. |
+
+
+## Policy Diagnosers
+
+Scope: `spec.kafkaStreamsPolicy.diagnoser[i]`
+
+| Diagnoser Field Suffix <br/> Full Field Name: _spec.kafkaStreamsPolicy.diagnoser[i].SUFFIX_ | Description                                                                                                                                                                                                                                                                                                                                                                             |
+|---------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| type                                                                                        | Specifies the type of this diagnoser. One of PROCESSING_RATE_SCALE_UP, PROCESSING_RATE_SCALE_DOWN, EXPECTED_LATENCY, or THREAD_SATURATION.                                                                                                                                                                                                                                              |
+| expectedLatency                                                                             | Specifies an Expected Latency Diagnoser. This Diagnoser specifies a goal for the expected latency for a record to be processed at a sub-topology, when enqueued at a source topic partition of that sub-topology.                                                                                                                                                                       |
```

Comment:
we should probably split the different diagnosers into their own table (e.g. one for `expectedLatency`, one for `threadSaturation`, etc)

Reply:
good call, will do